fenix/app/src/main/res/values/strings.xml

```diff
@@ -384,7 +384,7 @@
     <!-- Preference category for settings related to changing the default search engine -->
     <string name="preferences_category_select_default_search_engine">Select one</string>
     <!-- Preference for settings related to managing search shortcuts for the quick search menu -->
-    <string name="preferences_manage_search_shortcuts">Manage search shortcuts</string>
+    <string name="preferences_manage_search_shortcuts">Manage alternative search engines</string>
```

Comment:
When updating a string, we need to change the ID so that our localization tools pick it up correctly. Docs [here](https://github.com/mozilla-mobile/firefox-android/blob/main/fenix/docs/Working-with-Strings.md) for reference.

```suggestion
    <string name="preferences_manage_search_shortcuts2">Manage alternative search engines</string>
```